students/JaeKim/lesson04/assignment/tests/test_basic_operations.py

```diff
@@ -0,0 +1,202 @@
+import pytest
+from assignment.customers_model import Customer, TestCustomer
+import create_customers as cc
+from assignment.basic_operations import add_customer, search_customer, delete_customer, update_customer_credit, list_active_customers
+import csv
+
+cc.setup_db(environment='dev')
+
+"""
+Module docstring
+"""
+#
+
+OK_CUSTOMER = {'customer_id': '12345',
+               'name': 'Suzie',
+               'lastname': 'Edgar',
+               'home_address': '123 Browncroft Blvd, Rochester, NY 97235',
+               'phone_number': '234-123-4567',
+               'email_address': 'suzie@hotmail.com',
+               'status': True,
+               'credit_limit': 40}
+
+
+def import_users():
+    data = list()
+    try:
+        with open('data.csv', 'r') as csvfile:
+            spamreader = csv.reader(csvfile, delimiter=',')
+            for row in spamreader:
+                dict_user = dict()
+                # customer_id,name,last_name,home_address,phone_number,email_address,status,credit_limit
+                dict_user['customer_id'] = row[0]
+                dict_user['name'] = row[1]
+                dict_user['lastname'] = row[2]
+                dict_user['home_address'] = row[3]
+                dict_user['phone_number'] = row[4]
+                dict_user['email_address'] = row[5]
+                dict_user['status'] = row[6]
+                dict_user['credit_limit'] = row[7]
+                data.append(dict_user)
```

Comment:
This looks like an excellent place for a dictionary comprehension. :) Also, this function should be in basic_operations, because tests aren't the only place you might want to add customers from a csv file. Then, finally, you might want to think about making this function a generator. Then you don't have to do the iter call down below.